config/helm/appmesh-controller/values.yaml

```diff
@@ -106,8 +106,10 @@ tracing:
   provider: x-ray
   # tracing.address: Jaeger or Datadog agent server address (ignored for X-Ray)
   address: appmesh-jaeger.appmesh-system
-  # tracing.port: Jaeger or Datadog agent server port
+  # tracing.port: X-Ray, Jaeger or Datadog agent server port
   port: 2000
+  # tracing.samplingRate: X-Ray tracer sampling rate
+  samplingRate: ""
```

Comment:
Shouldn't this be set to 0.05 ?

Reply:
Envoy X-Ray extension has 3 types of sampling strategy.
- Default -> 5%
- Localized -> custom json file needed in envoy container
- Dynamic -> Support pending

When we configure this `samplingRate` variable it will create a **Localized** sampling rule file and sets sampling rate provided in this variable.

If we just want to go with the defaults then there is no to create this localized sampling rules file.

Reply:
So now for readability I will set this samplingRate as 0.05 itself. In envoy bootstrap generation script we will handle if we block creating the localized sampling rule manifest file.